go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/ahmetb/go-linq
```

Comment:
you sure this shouldn't have the prefix .../go-linq<b>/v3</b>?

Reply:
It should yes